signer/src/transaction_signer.rs

```diff
@@ -463,7 +465,7 @@ where
         let multi_sig = MultisigTx::new_tx(&request.contract_call, &wallet, request.tx_fee);
         let txid = multi_sig.tx().txid();
 
-        // TODO: Make this more robust. The signer that recieves this won't
+        // TODO: Make this more robust. The signer that receives this won't
```

Comment:
If there's a ticket for this can you link it like `TODO(#12345):`, and if there isn't a ticket can you very quickly make a `tech debt` ticket? It should be fast, you can just put minimal details into it but we don't want to lose track.

Reply:
We don't. And I should remove or reword this TODO.

I'm leaning toward removing the digest field since it will be unnecessary if we just switch to signing the entire message bytes. This work is basically https://github.com/stacks-network/sbtc/issues/517, and is super easy to do. In fact, I will do it now.

Reply:
Actually, I won't do this now, and just update the todo.

Reply:
Added a TODO and reworded this in https://github.com/stacks-network/sbtc/pull/694/commits/c3ceea7f8a43bcdd8ef975170e6c5d9aaf965461.